src/main/java/com/shigotosagashi/jobdomainservice/repository/JobEntityRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.shigotosagashi.jobdomainservice.repository;
+
+import com.shigotosagashi.jobdomainservice.domain.JobEntity;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface JobEntityRepository extends CrudRepository<JobEntity, String> {
+
+  List<JobEntity> findByTitleLike(String title);
+  List<JobEntity> findByDescriptionLike(String description);
+  List<JobEntity> findByExpirationDateAfter(long datetime);
```

Comment:
We're not actually using any of these methods. I would remove them.
And when we go to real production, we won't execute "like" queries in database anyway.